lib/pymedphys/_experimental/pinnacle/image.py

```diff
@@ -226,5 +226,5 @@ def convert_image(image, export_path):
             export_path, f"{image.image['Modality']}.{imageds.SOPInstanceUID}.dcm"
         )
 
-        imageds.save_as(output_file)
+        imageds.save_as(output_file, False)
```

Comment:
Would this be able to call False as a keyword argument? Ends up being easier to read if it is quite obvious what False is being assigned to.

Reply:
Good Idea, I'll submit the amendment